### ISSUE TITLE
Doesn't work in a monorepo if the `package.json` in the root of the project doesn't have a "name" property

### ISSUE THREAD

**Issue by TheKnarf** [webpro/knip#539]
```
npx knip
Reading workspace configuration(s)...
Missing package name in (project directory)/package.json
```

I'm not sure why I would have to set a "name" property in the `package.json` in the root of my project. As no other tools require this.

**Comment by webpro** [comment]
Since Knip requires/uses a root workspaces, and it can be targeted in features like `--directory`, there must be a name. Also for features like #538.

**Comment by TheKnarf** [comment]
It should still work if the root `package.json` doesn't have a name. It's very uncommon in a monorepo for the root to have the name set.

**Comment by webpro** [comment]
If I'm not mistaken, you're the first to mention this. Feel free to open a PR, happy to review!

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v5.1.2. See [Release 5.1.2](https://github.com/webpro/knip/releases/tag/5.1.2) for release notes._

**Comment by webpro** [comment]
Alright, here we go :)
